### ISSUE TITLE
Input not required

### ISSUE THREAD

**Issue by klloyd413** [foxysolutions/LWC-RecordtType-PicklistValue-Selector#6]
While testing my screen flow which is using this component, the input of the fields are not required upon creation of the record. I checked the "Require Input" option and it is set to GlobalConstant.True. Is there something I am missing? I can not move this to production without this resolved and it is expected to deploy in a week.

**Comment by foxysolutions** [comment]
@klloyd413 yes this is unfortunately a known issue, as can be found in the comments on [my blog](https://www.foxy-solutions.com/salesforce/recordtype-restricted-picklist-values-in-flow/). The component makes use of the standard `lightning-input-field` and so far I've understood there is no error thrown if the handleFieldChange() is not called (so if someone doesn't make a change). This is from UI perspective a really weird behaviour, but something I can't change unfortunately.
A possible workaround is add a decision after the component to validate and if not filled, set an assignment on failure and redirect to the screen where the error is conditionally shown.

I'll check if I can find any update from Salesforce on this, but am afraid this is the best approach to meet your timelines.
Please let me know if this resolves the issue, or whether you feel I might have misinterpreted the issue, if so please share some screenshots to clarify.

Good luck!
